### DISCUSSION TITLE
What is the licence of river?

### DISCUSSION THREAD

**Discussion by kync** [riverqueue/river#181] [Q&A] [answered]
I see the current licence is MPL 2.0, but I also see it has GPL.txt in the repo which confuses me. Would you clarify what is the licence of river? If it is MPL, is there any reason you are keeping GPL.txt?

**Comment by bgentry** [comment, marked as answer]
It’s MPLv2, looks like I missed some remnants of LGPL when switching the repo. I've fixed those in https://github.com/riverqueue/river/discussions/181, thank you!
